index.html

```diff
@@ -123,7 +123,7 @@
         <p>A deck which contains only Standard cards is called a <i>Standard deck</i>.</p>
         <h3>
           Banlist
-          <a class="tip" title="Despite being printed into Standard sets, these cards are not allowed in Standard decks as of January 2017."></a>
+          <a class="tip" title="Despite being printed into Standard sets, these cards are not allowed in Standard decks as of April 2017."></a>
```

Comment:
I'd prefer to remove the "as of blah" part of this sentence now, as this could accidentally be read as "April 2017 is when these cards became banned".